src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -640,19 +641,22 @@ public function getScriptURL()
         // If gzip is disabled just return core script url
         $useGzip = HTMLEditorField::config()->get('use_gzip');
         if (!$useGzip) {
-            return ADMIN_THIRDPARTY_DIR . '/tinymce/tinymce.min.js';
+            return $this->getTinyMCEPath() . '/tinymce.min.js';
         }
 
         // tinyMCE JS requirement
-        require_once ADMIN_THIRDPARTY_PATH . '/tinymce/tiny_mce_gzip.php';
-        $tag = TinyMCE_Compressor::renderTag(array(
-            'url' => ADMIN_THIRDPARTY_DIR . '/tinymce/tiny_mce_gzip.php',
-            'plugins' => implode(',', $this->getInternalPlugins()),
-            'themes' => $this->getTheme(),
-            'languages' => $this->getOption('language')
-        ), true);
-        preg_match('/src="([^"]*)"/', $tag, $matches);
-        return html_entity_decode($matches[1]);
+        if (file_exists($this->getTinyMCEPath() . '/tiny_mce_gzip.php')) {
```

Comment:
getTinyMCEPath() is a relative path (e.g. `silverstripe-admin/thirdparty/tinymce`). If you want to do filesystem checks you need to convert this to full path. Combine with BASE_PATH to do this.